app/helpers/application_helper.rb

```diff
@@ -24,6 +24,45 @@ def miq_accordion_panel(title, condition, id, &block)
     end
   end
 
+  def single_relationship_link(record, table_name, property_name = nil)
+    out = ''
+    property_name ||= table_name
+    ent = record.send(property_name)
+    name = ui_lookup(:table => table_name.to_s)
+    if role_allows(:feature => "#{table_name}_show") && !ent.nil?
+      out = content_tag(:li) do
+        link_to("#{name}: #{ent.name}",
+                {:controller => table_name, :action => 'show', :id => ent.id.to_s},
+                 :title => _("Show this %{entity_name}'s parent %{linked_entity_name}") %
+                           {:entity_name        => record.class.name.demodulize.titleize,
+                            :linked_entity_name => name}
+        )
+      end
+    end
+    out
+  end
+
+  def multiple_relationship_link(record, table_name)
```

Comment:
I think it would be better to add options such as a custom ```feature```.
For example, the feature for ```containers``` is "containers" and not ```container_show_list```
you can see that in ```container_summary_helper.rb``` the method for containers is ```textual_link(@record.containers, :feature => "containers")```

I would model this after ```textual_summary_helper.rb``` since that stabilized and this isnt so different functionally.